### PR TITLE
PADV-1182 feat: add instructor profile

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-1182

### Description
Add instructor profile

### Changes made
Add Instructor profile page
Add api to get instructor data
Add test

### Screenshots
Screenshot from 2024-08-15 16-21-22

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1990/my-profile

### CODE REVIEW

File: package.json
Comment (01001110J) [inline package.json:1]: Please do not forget to upgrade to the latest version of `react-paragon-topaz`

File: src/features/Instructor/Profile/index.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  const instructorUserName = useSelector((state) => state.main.username);
+  const classes = useSelector((state) => state.common.allClasses.data);
+
+  const isLoading = status === RequestStatus.LOADING;
+
+  useEffect(() => {
+    if (instructorEmail) {
+      dispatch(fetchInstructorProfile(instructorEmail));
+      dispatch(fetchAllClassesData(instructorUserName));
+    }
```
Comment (01001110J) [inline src/features/Instructor/Profile/index.jsx:40]: What would happen if I made changes using the institution selector?

File: src/features/Instructor/data/slice.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { RequestStatus } from 'features/constants';
+
+const initialState = {
+  instructorInfo: {
```
Comment (01001110J) [inline src/features/Instructor/data/slice.js:6]: Not sure about this, is kinda redundant `instructor.instructorInfo` I would consider to change it to `instructor.info`

File: src/features/Instructor/Profile/index.jsx (hunk 1 above)
Comment (AuraAlba) [reply src/features/Instructor/Profile/index.jsx:40]: That is the reason I needed your changes

Comment (01001110J) [reply src/features/Instructor/Profile/index.jsx]: a

**Review by 01001110J — APPROVED** [review]

**Pull request merged**